release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -550,6 +550,21 @@ In this release you can now specify the transmit hash policy for load balancing
 
 For more information, see xref:../networking/multiple_networks/secondary_networks/creating-secondary-nwt-other-cni.adoc#nw-multus-bond-cni-object_configuring-additional-network-cni[Configuration for a Bond CNI secondary network].
 
+[id="ocp-4-20-networking-namespaced--sriov-app-owners_{context}"]
+==== Directly manage your SR-IOV Networks in your application namespace
+
+This release introduces namespaced SR-IOV networks, allowing application owners to directly create and manage their own SR-IOV networks within their application namespaces. This feature gives you greater control over your network configurations and helps simplify your workflow. 
+
+Previously, creating an SR-IOV network required a cluster administrator to configure it for you. Now, you can manage these resources directly in your own namespace, which offers several key benefits:
+
+* Increased autonomy and control: You can now create your own SriovNetwork objects, removing the need to involve a cluster administrator for network configuration tasks. 
+
+* Enhanced security: Managing resources within your own namespace improves security by providing better separation between applications and helps prevent unintentional misconfigurations.
+
+* Simplified permissions: This new approach streamlines your permissions and reduces operational overhead.
```

Comment:
"This new approach" is a bit ambiguous IMO. Consider stating the feature to remove ambiguity, for example,  "Using namespaced SR-IOV networks streamlines...."